app/src/main/java/org/schulcloud/mobile/utils/WebUtils.kt

```diff
@@ -46,6 +46,13 @@ fun String?.asUri(): Uri {
         Uri.parse(this)
 }
 
+fun String?.asPrefixedUri(): Uri {
```

Comment:
Maybe this functionality could be integrated in the general `asUri` function for all URIs without protocol?